test/libsolidity/semanticTests/inlineAssembly/basefee_berlin_function.sol

```diff
@@ -0,0 +1,22 @@
+contract C {
+    function f() public view returns (uint ret) {
+        assembly {
+            let basefee := sload(0)
+            ret := basefee
+        }
+    }
+    function g() public pure returns (uint ret) {
+        assembly {
+            function basefee() -> r {
+                r := 1000
+            }
+            ret := basefee()
+        }
+    }
+}
+// ====
+// compileViaYul: also
+// EVMVersion: =berlin
```

Comment:
If you change the other one to <= you can do this one as well.

Reply:
Also fixed.